functions/merge_sorted_lists.py

```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
+
+    return sorted(merged)
```

Comment:
I like this short and simple solution, Ying. How can we do this without using the sorted() helper function?